File: Web/src/java/web/Events.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package web;

import java.sql.Time;
import java.util.Date;

/**
 *
 * @author Johan Bos
 */
public class Events {
    private long unitId;
    private String port;
    private boolean value;
    private Date date;
    private Time time;

    public Events() {
    }

    public Events(long unitId, String port, boolean value, Date date, Time time) {
        this.unitId = unitId;
        this.port = port;
        this.value = value;
        this.date = date;
        this.time = time;
    }

    public long getUnitId() {
        return unitId;
    }

    public void setUnitId(long unitId) {
        this.unitId = unitId;
    }

    public String getPort() {
        return port;
    }

    public void setPort(String port) {
        this.port = port;
    }

    public boolean isValue() {
        return value;
    }

    public void setValue(boolean value) {
        this.value = value;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public Time getTime() {
        return time;
    }

    public void setTime(Time time) {
        this.time = time;
    }

    @Override
    public String toString() {
        return "Events{" + "unitId=" + unitId + ", port=" + port + ", value=" + value + ", date=" + date + ", time=" + time + '}';
    }
    
    
           
}

File: DataClient/Service/Events.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Service
{
    public class Events
    {
        public virtual long UnitId { get; set; }
        public virtual string Port { get; set; }
        public virtual Boolean Value { get; set; }
        public virtual DateTime Date { get; set; }
        public virtual DateTime Time { get; set; }

        public Events() { }

        public Events(long UnitId, string Port, Boolean Value, DateTime Date, DateTime Time)
        {
            this.UnitId = UnitId;
            this.Port = Port;
            this.Value = Value;
            this.Date = Date;
            this.Time = Time;
        }

        public override string ToString()
        {
            return String.Format("Event: {0}, Unit ID:{1}, Port:{2}, Value:{3}, Date{4}, Time{5}", UnitId, Port, Value, Date, Time);
        }
    }
}